content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -70,9 +70,9 @@ Any commands under `kubeadm alpha` are, by definition, supported on an alpha lev
 
 ## Instructions
 
-### Installing kubeadm on your hosts
+### Preparing the hosts
 
-See ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
+Install {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}  and kubeadm on all the hosts. For detailed instructions and other prerequisites, see ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```

Comment:
```suggestion
Install a {{< glossary_tooltip term_id="container-runtime" text="container runtime" >}} and kubeadm
on all the hosts. For detailed instructions and other prerequisites, see ["Installing kubeadm"](/docs/setup/production-environment/tools/kubeadm/install-kubeadm/).
```

- broke it down into 2 lines
- added missing `a` before `container runtime`
- removed extra space before `and kubeadm`

thanks for the PR, the change SGTM.